compiler/rustc_builtin_macros/src/test.rs

```diff
@@ -396,6 +401,12 @@ fn should_ignore_message(cx: &ExtCtxt<'_>, i: &ast::Item) -> Option<Symbol> {
     }
 }
 
+fn item_location_info(cx: &ExtCtxt<'_>, item: &ast::Item) -> Symbol {
+    let ident_sp = cx.with_def_site_ctxt(item.ident.span);
```

Comment:
I don't think it's necessary to set the def site context here, `span_to_string` doesn't use it.

Reply:
done.